auth0/src/test/java/com/auth0/android/authentication/storage/CredentialsManagerTest.java

```diff
@@ -356,12 +457,13 @@ public void shouldGetAndSuccessfullyRenewExpiredCredentials() {
         when(storage.retrieveString("com.auth0.scope")).thenReturn("scope");
         when(client.renewAuth("refreshToken")).thenReturn(request);
 
-        Date newDate = new Date(123412341234L);
+        Date newDate = new Date(CredentialsMock.CURRENT_TIME_MS + 1234 * 1000);
```

Comment:
The numeric value could be extracted as a `validExpirationTime` constant, as it's used in a few places.